examples/gno.land/r/demo/groups/group.gno

```diff
@@ -0,0 +1,131 @@
+package groups
+
+import (
+	"gno.land/p/demo/avl"
+	"std"
+	"strconv"
+	"time"
+)
+
+type GroupID uint64
+type MemberID uint64
+
+func (gid GroupID) String() string {
+	return strconv.Itoa(int(gid))
+}
+
+func (mid MemberID) String() string {
+	return strconv.Itoa(int(mid))
```

Comment:
Similar to this: https://github.com/gnolang/gno/pull/388/files#r1061834746
> Consider using:
> ```go
> strconv.FormatUint(gid, 10))
> ```
> or 
> ```go
> fmt.Sprintf("%v", gid)
> ```

Reply:
Just to piggyback here, @pushkar803 's `strconv.Itoa` seems to be much faster than regular string formats (ignore the year):
https://gist.github.com/evalphobia/caee1602969a640a4530

so it should be good to go for this use-case

Reply:
I was just worried about the conversion of 32-bit to 64-bit. What are your thoughts on `strconv.FormatUint`?

Reply:
done. kept strconv.FormatUint(gid, 10))